apps/api/src/app/integrations/e2e/get-integration.e2e.ts

```diff
@@ -0,0 +1,36 @@
+import { IntegrationEntity } from '@notifire/dal';
+import { UserSession } from '@notifire/testing';
+import { expect } from 'chai';
+
+describe('Get Integration - /integrations (GET)', function () {
```

Comment:
```suggestion
describe('Get Integrations - /integrations (GET)', function () {
```

Also suggest renaming the final name to plural